rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt

```diff
@@ -652,43 +652,49 @@ internal fun SymbolicConstantsContext.toAst(conf: ToAstConfiguration = ToAstConf
 internal fun Cspec_fixedContext.toAst(conf: ToAstConfiguration = ToAstConfiguration()): Statement {
     return when {
         this.cspec_fixed_standard() != null ->
-            this.cspec_fixed_standard().toAst(conf)
-                .also {
-                    it.indicatorCondition = this.toIndicatorCondition(conf)
-                    if (it.indicatorCondition != null) {
-                        val continuedIndicators = this.cspec_continuedIndicators()
-                        // loop over continued indicators (WARNING: continuedIndicators not contains inline indicator)
-                        for (i in 0 until continuedIndicators.size) {
-                            val indicator = continuedIndicators[i].indicators.children[0].toString().toIndicatorKey()
+            // we need capture error inside runParserRuleContext in order
+            // to avoid that some errors pass silently
+            this.cspec_fixed_standard().runParserRuleContext(conf) { standardContext ->
```

Comment:
Could you explain me this block of code about indicators conditions?

Reply:
I copy and paste the function:
```kotlin
/**
 * Run a block. In case of error throws an error encapsulating useful information
 * like node position
 */
fun <T : ParserRuleContext, R> T.runParserRuleContext(conf: ToAstConfiguration, block: (T) -> R): R {
    return kotlin.runCatching {
        block.invoke(this)
    }.onFailure {
        // to avoid duplicated errors
        if (it !is ParseTreeToAstError) this.error(it.message, it, conf)
    }.getOrThrow()
}
```
In addition the `this.error(it.message...)` statement - you can see the implementation from your ide - allows you to capture the error and fire it as ErrorEvent handled from a consumer (for example jardis) which implements the callback `JarikoCallback.onError` function, pratically if you see syntax errors in jardis is thanks to these stuff.